siliconcompiler/scheduler.py

```diff
@@ -112,11 +112,6 @@ def _defernode(chip, step, index):
             sf.write('#!/bin/bash\n')
             sf.write(f'sc -cfg {shlex.quote(cfg_file)} -builddir {shlex.quote(buildir)} '
                      f'-arg_step {shlex.quote(step)} -arg_index {shlex.quote(index)}\n')
-            # In case of error(s) which prevents the SC build script from completing, ensure the
-            # file mutex for job completion is set in shared storage. This lockfile signals the
-            # server to mark the job done, without putting load on the cluster reporting/accounting
-            # system.
-            sf.write(f'touch {os.path.dirname(output_file)}/done')
```

Comment:
Seems like unrelated change. Why is this being removed?

Reply:
That was to make debugging easier and is now merged: https://github.com/siliconcompiler/siliconcompiler/pull/1950